### DISCUSSION TITLE
ajout des actifs de type IOT

### DISCUSSION THREAD

**Discussion by moiiom59** [dbarzin/mercator#507] [General]
Serait il possible d'ajouter des ressources de type IOT dont on pouurait identifier les attributs (Non , Constructeur, IP, MAC, Description, Protocoles, Version firmware, ...)
merci

**Comment by dbarzin** [comment]
Vous pouvez utiliser la table "périphériques" pour gérer les IOT. Les champs suivants sont présents : 
- nom
- description
- type
- localisation

On peut ajouter d'autres champs et même un lien avec les applications qui l'utilise.

**Comment by moiiom59** [comment]
Bonjour Monsieur Barzin,
Parfait, si vous pouvez ajouter des champ comme le domaine du Périphérique ou nous pourrions spécifier qu’il s’agit d’un IOT, le fabriquant.
Avoir un lien avec un référent (personne), une application


  *   nom
  *   description
  *   fabriquant
  *   type
  *   localisation
  *   domaine (IT, OT, IOT)
Lien Personne, Application
Ce serait parfait
Bien cordialement,

**Reply by dbarzin** [reply]
Pour le fabriquant, faisons-nous le lien avec la table "entités" ? Cela permet d'avoir les points de contact et les relations.
De quelle personne parlons-nous ?

**Comment by moiiom59** [comment]
Monsieur Barzin ,
Le table périphérique reprend (Nom, Type, Responsable, Site, Bâtiment / Salle, Baie)
Et donc il serait intéressant d’ajouter le Fabriquant , le domaine (IT, OT, IOT) et le lien sur une application
Merci à vous

Bien cordialement,

**Comment by moiiom59** [comment]
Le fabriquant dans entité OK
Et pour la personnel on a le responsable donc OK
La différenciation entre les périphériques IT, OT , IOT reste intéressante

merci


Bien cordialement,

**Comment by dbarzin** [comment]
OK, ce sera disponible rapidement.

**Comment by moiiom59** [comment]
Pourrait on avoir accès à cette listes ?
***@***.***

**Reply by dbarzin** [reply]
C'est ici : https://github.com/dbarzin/mercator/discussions/508

**Comment by dbarzin** [comment]
C'est disponible
